onnxruntime/core/providers/openvino/openvino_execution_provider.cc

```diff
@@ -201,7 +201,7 @@ bool IsOpSupported(std::string name){
 
 //Checks if the entire graph is supported by OpenVINO EP and returns false if it is not.
 
-bool IsGraphSupported(const onnxruntime::GraphViewer& graph_viewer, std::string dev_id){
+void IsGraphSupported(const onnxruntime::GraphViewer& graph_viewer, std::string dev_id){
```

Comment:
Re-name to 'CheckGraphSupported' for code cleanliness as we won't use it inside an if condition anymore.